src/Fantomas/CodePrinter.fs

```diff
@@ -1926,7 +1926,7 @@ and genExpr astContext synExpr ctx =
                 genExpr astContext e
                 +> sepSpace
                 +> sepOpenTFor lpr
-                +> col sepComma args (genShortExpr astContext)
+                +> col sepComma args (genShortExpr astContext >> genTriviaFor SynExpr_Tuple rangeOfTuple)
```

Comment:
Here you are printing the trivia of the tuple for each item inside the tuple.
What you want is something like:
```fsharp
(col (sepComma +> sepNln) args (genExprLong astContext) |> genTriviaFor SynExpr_Tuple rangeOfTuple)
```

Reply:
Oh I see! Thanks for the info.